h2o-algos/src/test/java/hex/glm/GLMdispersionFactorTest.java

```diff
@@ -0,0 +1,52 @@
+package hex.glm;
+
+import org.junit.Test;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Frame;
+
+import static hex.glm.GLMModel.GLMParameters.DispersionMode.ml;
+import static hex.glm.GLMModel.GLMParameters.DispersionMode.pearson;
+import static hex.glm.GLMModel.GLMParameters.Family.gamma;
+import static hex.glm.GLMModel.GLMParameters;
+
```

Comment:
Isn't this part missing here?

```
@RunWith(H2ORunner.class)
@CloudSize(1)
```

Reply:
yes, the test is not going to work properly in isolation - @wendycwong you need to either do what Zuzana is suggesting or make sure H2O is correctly initialized using @BeforeClass

Reply:
Yes, you guys are RIGHT!  Added!